client/layout/universal-navbar-footer/index.tsx

```diff
@@ -11,6 +11,7 @@ import { navigate } from 'calypso/lib/navigate';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 
 const UniversalNavbarFooter = () => {
+	const isEnglishLocale = useIsEnglishLocale();
 	const translate = useTranslate();
 	const localizeUrl = useLocalizeUrl();
 	const locale = getLocaleSlug() ?? undefined;
```

Comment:
This incorrect locale value seems to be used in a few `localizeUrl` calls, which may cause unexpected results on the SSR. However, that seems to be out of the scope of this PR so we should probably address it in a separate PR.